src/components/AuthButton.jsx

```diff
@@ -1,11 +1,11 @@
-import React, { PropTypes } from 'react';
+import { PropTypes } from 'react';
```

Comment:
@eatyourgreens @rogerhutchings Think I've figured it out. The reason this component still works is that I have exposed `React` in the `window` object.